docs/feature_oled_driver.md

```diff
@@ -399,7 +399,7 @@ void oled_scroll_set_area(uint8_t start_line, uint8_t end_line);
 // Sets scroll speed, 0-7, fastest to slowest. Default is three.
 // Does not take effect until scrolling is either started or restarted
 // the ssd1306 supports 8 speeds with the delay
-// listed below betwen each frame of the scrolling effect
+// listed below between each frame of the scrolling effect
```

Comment:
Same error is in `drivers/oled/oled_driver.h` as its copy-pasted from that file.